src/proteus/utils/constants.py

```diff
@@ -54,19 +54,23 @@
 element_list = ["H", "O", "C", "N", "S", "Si", "Mg", "Fe", "Na"]
 
 ## Constant from Zephyrus
-ergcm2stoWm2  = 1e-3                  # convert [erg s-1 cm-2] to [W m-2]
-s2yr          = 1/(3600*24*365)       # convert [seconds]      to [years]
+ergcm2stoWm2        = 1e-3              # convert [erg s-1 cm-2] to [W m-2]
+s2yr                = 1/(3600*24*365)   # convert [seconds]      to [years]
 # Sun parameters
-Rs      = 6.957e8                      # Solar radius                          [m]
-Ms      = 1.98847e30                   # Solar mass                            [kg]
-age_sun = 4.603e9                      # Age of the Sun                        [yr]
+Rs                  = 6.957e8       # Solar radius [m]
+Ms                  = 1.98847e30    # Solar mass [kg]
+age_sun             = 4.603e9       # Age of the Sun [yr]
```

Comment:
```suggestion
age_sun             = 4.568e9       # Age of the Sun [yr]
```

This number was wrong.